creating_gs_app.md

```diff
@@ -29,46 +29,76 @@ to put it, we have created a `guest_science_projects` folder for you to use.
 Please note, you don't have to use this directory and can store your project
 anywhere on your computer.
 
+**Important**: Since the Astrobee team is currently in the process of transitioning from 
+Ubuntu 16.04 to 20.04, there will be a divergence in the procedure to accomadate for both Ubuntu versions. Follow the guide according to your version and assume if there is no indication for which version it is for, that the steps work for both.
+
 ## Setup
 
 In every terminal you use, be sure to set up your environment. If you forgot how
 to do this, follow the Setting up your Environment section of the
-[`freeflyer/simulation/sim_overview.md`](https://github.com/nasa/astrobee/blob/master/simulation/sim_overview.md#setting-up-your-environment).
+[`astrobee/simulation/running_the_sim.md`](https://github.com/nasa/astrobee/blob/master/simulation/running_the_sim.md).
 
 ## Build the Astrobee API
 
 ### Generating ARS ROS Messages (`ff_msgs`)
 
-Ensure `rosjava` is installed:
-
-    you@machine:~ $ sudo apt-get install ros-kinetic-rosjava
+**Ubuntu 16.04**
 
+Ensure `rosjava` is installed:
+```shell
+  sudo apt-get install ros-kinetic-rosjava
+```
 Build the ff_msgs jar
-
-    you@machine:~ $ cd $BUILD_PATH
-    you@machine:native $ make rebuild_cache
-    you@machine:native $ make ff_msgs_generate_messages_java_gradle
-
+ <!--- - Note: You may not need to build the ff_msgs jar, as they should be built when you install the Astrobee Software. In the event that they weren't, run the commands below. -->
+ 
+ <!--- This is something I was told, and so not to mislead people I will leave it out -->
+ 
+```shell
+  cd $BUILD_PATH
+  make rebuild_cache
+  make ff_msgs_generate_messages_java_gradle
+```
+**Ubuntu 20.04**
+
+As there is no ros-noetic-java for Ubuntu 20.04, the current solution is for
+developers to download the jar files here:
+
+https://github.com/nasa/astrobee/actions/workflows/msgs_jar.yaml?query=branch%3Adevelop
```

Comment:
If you want to make this look nicer, you can use a Github badge instead of a plain link. Like this:

[![Build jar messages](https://github.com/nasa/astrobee/actions/workflows/msgs_jar.yaml/badge.svg?branch=develop)](https://github.com/nasa/astrobee/actions/workflows/msgs_jar.yaml)

Let me know if you have questions about it

Reply:
After a while I finally figured out how to make it, also I have it so that it goes to the branch:develop as that was the one you showed me. Hope that's okay with you.